connectors/sources/mssql.py

```diff
@@ -100,15 +93,14 @@ def _create_engine(self):
         """Create sync engine for mssql"""
         if self.secured_connection:
             query = {
-                "driver": self.mssql_driver,
                 "TrustServerCertificate": "no",
```

Comment:
Regarding `TrustServerCertificate` and `Encrypt` it's specific to odbc driver. so I think we need to change this configuration.